go/common/consensus/genesis/genesis.go

```diff
@@ -0,0 +1,12 @@
+// Package consensus provides consensus config flags that should be part of the genesis state.
+package consensus
```

Comment:
So this has been moved to `consensus/genesis` so it should be `genesis` now probably? Is the reason for the move some issue with circular imports?

Reply:
yes and yes :)